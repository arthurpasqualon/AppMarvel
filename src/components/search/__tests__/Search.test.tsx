import React, {useState} from 'react';
import renderer from 'react-test-renderer';
import Search from '../index';

function SearchMock() {
  const [text, setText] = useState('');
  return (
    <Search
      onSubmitEditing={() => {}}
      label="Label"
      value={text}
      setValue={setText}
    />
  );
}
test('renders correctly', () => {
  const tree = renderer.create(<SearchMock />).toJSON();
  expect(tree).toMatchSnapshot();
});
