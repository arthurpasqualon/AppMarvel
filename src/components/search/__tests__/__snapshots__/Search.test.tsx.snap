// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flexDirection": "row",
        "marginBottom": 5,
        "width": "100%",
      },
    ]
  }
>
  <TextInput
    allowFontScaling={true}
    onChangeText={[Function]}
    onSubmitEditing={[Function]}
    placeholder="Label"
    placeholderTextColor="#E5E5E5"
    rejectResponderTermination={true}
    style={
      Array [
        Object {
          "color": "#ffffff",
          "fontFamily": "RobotoCondensed-Bold",
          "fontSize": 24,
          "marginBottom": 10,
          "marginLeft": "7%",
          "marginRight": 10,
          "marginTop": 10,
          "maxWidth": "80%",
        },
      ]
    }
    underlineColorAndroid="transparent"
    value=""
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignSelf": "center",
        "opacity": 1,
        "resizeMode": "contain",
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/assets/search.png",
        }
      }
      style={
        Array [
          Object {
            "height": 22,
            "width": 22,
          },
        ]
      }
    />
  </View>
</View>
`;
